models.go

```diff
@@ -49,20 +49,28 @@ type FeatureStateValue struct {
 }
 
 type FeatureState struct {
-	ID                int64              `json:"id"`
+	ID                int64              `json:"id,omitempty"`
+	UUID              string             `json:"uuid,omitempty"`
 	FeatureStateValue *FeatureStateValue `json:"feature_state_value"`
 	Enabled           bool               `json:"enabled"`
 	Feature           int64              `json:"feature"`
-	Environment       int64              `json:"environment"`
+	Environment       *int64             `json:"environment"`
```

Comment:
Just checking here - it looks like we've allowed Environment to be nullable, but it should never be null, right?

Reply:
Yeah, It can be because we create/import feature states from Terraform using the environment key